server/src/main/java/io/crate/protocols/postgres/PostgresWireProtocol.java

```diff
@@ -299,19 +299,24 @@ public void channelRegistered(ChannelHandlerContext ctx) throws Exception {
         @Override
         public void channelRead0(ChannelHandlerContext ctx, ByteBuf buffer) throws Exception {
             assert channel != null : "Channel must be initialized";
+            PGError error = null;
             try {
                 dispatchState(buffer, channel);
             } catch (Throwable t) {
                 ignoreTillSync = true;
                 if (session != null) {
-                    t = SQLExceptions.forWireTransmission(getAccessControl.apply(session.sessionContext()), t);
+                    // Change handling here
+                    error = PGError.fromThrowable(t, getAccessControl.apply(session.sessionContext()));
```

Comment:
Would it make sense to move this into the `sendErrorResponse` method, so we really only have a single place that takes care of the exception conversion?